.gitignore

```diff
@@ -7,6 +7,7 @@
 *.dll
 *.so
 *.dylib
+*.txt 

```

Comment:
We should use “.log” as a file extension.

Reply:
Alright